pkg/mfa/client.go

```diff
@@ -153,6 +153,16 @@ type RawVerifyResponse struct {
 	VerifyResponseError
 }
 
+type DeleteFactorOpts struct {
+	// ID of factor to be deleted
+	ID string
```

Comment:
It looks like the convention is to have a more descriptive name - could we use `AuthenticationFactorID` instead for both of these new methods?